### DISCUSSION TITLE
missing email notifications

### DISCUSSION THREAD

**Discussion by rejetto** [community/community#76966] [Discussions] [answered]
### Select Topic Area

Bug

### Body

recently i'm often missing notification for new issues and new discussions.
The problem seems random.
Am I the only one?

**Comment by NeekuroVonz** [comment, marked as answer]
for notification issue on certain repo, you have to select Watch > All Activity in the upper right, you should start getting notifications
![image](https://github.com/community/community/assets/47558978/9952bc5a-f41f-4415-b3d5-f30ee1be14e0)
also just for sure you can go into your setting after click your profile avatar on the upper right corner and check for any unchecked notification part.

**Reply by rejetto** [reply]
i guess that was it. Somehow I changed it, while sleeping, and my later checks were limited to "settings" area. I feel dumb.
Thanks
